### DISCUSSION TITLE
桌面出現圖示

### DISCUSSION THREAD

**Discussion by jasminegardenia** [SimonMacer/AnWave#34] [Open Discussion]
原本只是想要用用看，改變檔案總管界面成Windows 10版，結果桌面出現桌面、文件、下載、音樂、圖片及影片資料夾，還改不回來...

**Comment by SimonMacer** [comment]
> 原本只是想要用用看，改變檔案總管界面成Windows 10版，結果桌面出現桌面、文件、下載、音樂、圖片及影片資料夾，還改不回來...

不會改不回來，你應該是勾選了顯示這些資料夾的桌面圖示，請打開 Windows Key-Point > 個人化 > 在檔案總管中新增或刪除本機 (This PC) 下的資料夾群組中的 [顯示桌面圖示] 取消你不想要顯示的資料夾
<img width="375" alt="wksIcon" src="https://github.com/user-attachments/assets/760cd23f-9e03-49d6-a954-dff6d0de3e48">
